.env.example

```
@@ -24,5 +24,13 @@ RESEND_FROM_EMAIL=noreply@yourdomain.com
 RESEND_FROM_NAME=YourAppName
 RESEND_AUDIENCE_ID=<your-resend-audience-id>
 
+# PostHog (Analytics)
+NEXT_PUBLIC_POSTHOG_ACTIVATED=true
+NEXT_PUBLIC_POSTHOG_API_KEY=<your-posthog-api-key>
+NEXT_PUBLIC_POSTHOG_HOST=https://app.posthog.com
+NEXT_PUBLIC_POSTHOG_ENABLE_SESSION_REPLAY=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_ENABLE_SESSION_REPLAY key should go before the NEXT_PUBLIC_POSTHOG_HOST key

(UnorderedKey)